### ISSUE TITLE
NukeUI shows images despite `.redacted(reason: .placeholder)`

### ISSUE THREAD

**Issue by majelbstoat** [kean/Nuke#669]
I replaced my calls to `AsyncImage` with `LazyImage`and it works beautifully. 

On many of my List screens, while I'm waiting for the network request, I show a cheap placeholder which shows the view populated with some mock data, applying `.redacted(reason: .placeholder)`, e.g.:

```swift
struct QuestionListView {

    struct ContentView: View {
      var question: Question
      var body: some View {
        Text(question.title)
        LazyImage(url: question.imageURL)  
      }
    }

    struct Placeholder: View {
        var body: some View {
            QuestionListView.ContentView(questions: Question.mockData)
                .redacted(reason: .placeholder)
                .disabled(true)
        }
    }
}
```

When this was AsyncImage, the image was redacted as expected. But, when using LazyImage, the actual real image from the URL in the mockData is displayed. (It's hardcoded real-ish data.)

I'm not sure if this is a bug or a feature :) Is there an easy way to keep this capability?

**Comment by kean** [comment]
This is fixed in [version 12.0 (beta 1)](https://github.com/kean/Nuke/releases/tag/12.0.0-beta.1)

**Comment by majelbstoat** [comment]
This is amazing. Such a great project, thank you 🙏🏼
